src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/FindReferences/FindReferencesEndpointTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable disable
```

Comment:
Now that we have xunit versions with nullable knowledge I'd prefer new tests be `#nullable enable`.

Reply:
Oh awesome